Formula/handbrake.rb

```diff
@@ -1,8 +1,8 @@
 class Handbrake < Formula
   desc "Open-source video transcoder available for Linux, Mac, and Windows"
   homepage "https://handbrake.fr/"
-  url "https://download.handbrake.fr/releases/1.3.1/HandBrake-1.3.1-source.tar.bz2"
-  sha256 "c005de993af4123babbb60f7d12f6ff46fb7eea26ff0a3556034683901ce2c27"
+  url "https://github.com/HandBrake/HandBrake/releases/download/1.3.2/HandBrake-1.3.2-source.tar.bz2"
```

Comment:
Is this the official source now?

Reply:
@SMillerDev I was not sure -- the link `https://download.handbrake.fr/releases/1.3.2/HandBrake-1.3.2-source.tar.bz2` is HTTP 404, and their website points to the GitHub link I used:

<img width="723" alt="Screen Shot 2020-05-05 at 23 42 34" src="https://user-images.githubusercontent.com/6127163/81085734-304d6b80-8f2a-11ea-8ea2-b9663a939cab.png">